libfreerdp/codec/audio.c

```diff
@@ -38,7 +38,7 @@ UINT32 audio_format_compute_time_length(const AUDIO_FORMAT* format, size_t size)
 
 	if (format->wBitsPerSample)
 	{
-		const size_t samples = = (size * 8) / format->wBitsPerSample;
+		const size_t samples = (size * 8) / format->wBitsPerSample;
```

Comment:
This fixup should be in its previous commit.

Reply:
done